internal/routers/http/http_test.go

```diff
@@ -0,0 +1,30 @@
+package http_test
+
+import (
+	r "address-book/internal/routers/http"
+	"address-book/internal/usecase/repository"
+
+	"github.com/stretchr/testify/assert"
+
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+var (
+	repo   = repository.NewRepository()
+	router = r.SetupRouter(repo)
+	w      = httptest.NewRecorder()
```

Comment:
In golang, names should be clear for any who will read the code, especially if we are talking about global variables. What `w` means?
Useful link: https://github.com/golang/go/wiki/CodeReviewComments#variable-names